packages/@adobe/spectrum-css-temp/components/barloader/index.css

```diff
@@ -18,15 +18,27 @@ governing permissions and limitations under the License.
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: center;
-  width: var(--spectrum-barloader-large-width);
+  min-inline-size: 48px;
```

Comment:
DNA hasn't created a token so I'm using 48px. Is there a different token I should use?

Reply:
size-600 is 48px at medium

Reply:
used 600 static because they want 48px for medium and large scales.